test/test_common/utility.h

```diff
@@ -102,7 +103,10 @@ class TestUtility {
    * @param buffer supplies the buffer to convert.
    * @return std::string the converted string.
    */
-  static std::string bufferToString(const Buffer::Instance& buffer);
+  static std::string bufferToString(const Buffer::OwnedImpl& buffer) {
+    // TODO(jmarantz): remove this indirection and update all ~53 call sites.
```

Comment:
@jmarantz +1 can we do this as a cleanup sooner rather than later? Thank you.

Reply:
https://github.com/envoyproxy/envoy/pull/3736